README.md

```diff
@@ -71,15 +72,17 @@ Eclipse Collections is compatible with Java 8+. Eclipse Collections is a part of
 
 ## Eclipse Collections and JDK Compatibility Matrix
 
-| EC     | JDK 5 - 7 | JDK 8 | JDK 9 - 14 | JDK 15 - 21 |
-|--------|-----------|-------|------------|-------------|
-| 7.x.x  | &check;   |&check;|            |             |
-| 8.x.x  |           |&check;|            |             |
-| 9.x.x  |           |&check;| &check;    |             |
-| 10.x.x |           |&check;| &check;    |             |
-| 10.4.0 |           |&check;| &check;    | &check;     |
-| 11.x.x |           |&check;| &check;    | &check;     |
+| EC     | JDK 5 - 7 | JDK 8   | JDK 9 - 10 | JDK 11 - 14 | JDK 15 - 21 |
+|--------|-----------|---------|------------|-------------|-------------|
+| 7.x.x  | &check;   | &check; |            |             |             |
+| 8.x.x  |           | &check; |            |             |             |
+| 9.x.x  |           | &check; | &check;    |             |             |
+| 10.x.x |           | &check; | &check;    |             |             |
+| 10.4.0 |           | &check; | &check;    | &check;     | &check;     |
+| 11.x.x |           | &check; | &check;    | &check;     | &check;     |
+| 12.x.x |           |         |            | &check;     | &check;     |
```

Comment:
I can't exactly describe what's confusing me about this table, but the last two columns for JDK 11-14 and JDK 15-21 are identical, which makes them look like they can be collapsed. Looking at the old table, it's clear that 10.4.0 is the first version supporting Java 15+, but that doesn't show up in the new table. Maybe there's a mixxing check for 9.x.x and 10.x.x supporting JDK 11-14?

Reply:
You're correct, I forgot to carry over some checkmarks when I split JDK 9 - 14 up into 9-10 and 11-14